packages/foreman/rubygem-nokogiri/rubygem-nokogiri.spec

```diff
@@ -23,10 +23,6 @@ Provides:	bundled(gumbo-parser) = 0.10.1
 Requires: ruby >= 2.7.0
 BuildRequires: ruby-devel >= 2.7.0
 BuildRequires: rubygems-devel
-BuildRequires: rubygem(mini_portile2) >= 2.8.0
```

Comment:
We can move them out of the generated section to prevent them to be removed in the future. Like where we have `BuildRequires` on lib*-devel.

Reply:
It's the other way round. The gem has deps that we patch out ;-)

Reply:
Ah yes, that's harder.